helpers.lua

```diff
@@ -214,7 +214,7 @@ function delete_redirect_cookie()
                        "; Expires="..expired_time..
                        "; Secure"..
                        "; HttpOnly"..
-                       "; SameSite=Lax ;;"
+                       "; SameSite=Lax"
     ngx.header["Set-Cookie"] = "SSOwAuthRedirect=;" ..cookie_str
```

Comment:
> Syntax error in Set-Cookie: SSOwAuthRedirect=;; Path=/yunohost/sso/; Expires=Thu, 01 Jan 1970 00:00:00 UTC; Secure; HttpOnly; SameSite=Lax ;; at position 18.

I think we should remove this `;` here (or the one line 213 : `local cookie_str = "; Path="..conf["portal_path"]..`).